packages/web-react/src/components/Toast/README.md

```diff
@@ -263,13 +264,108 @@ const [isOpen, setIsOpen] = React.useState(false)
 </Toast>
 ```
 
-[web-toast]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/web/src/scss/components/Toast
-[mdn-role-log]: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/log_role
-[mdn-aria-live]: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live
+## UncontrolledToast
+
+```jsx
+import { ToastProvider, UncontrolledToast } from '@lmc-eu/spirit-web-react/components';
+```
+
+### Minimal props
+
+```jsx
+const { showToast } = useUncontrolledToast(); // must be inside ToastProvider
+
+<ToastProvider>
+  <Button type="button" onClick={() => showToast(ToastTextWithLink, 'toast-id')}>
+    Show Toast
+  </Button>
+
+  <UncontrolledToast />
+</ToastProvider>;
+```
+
+### All possible props
+
+```jsx
+const { showToast } = useUncontrolledToast(); // must be inside ToastProvider
+
+<ToastProvider>
+  <Button
+    type="button"
+    onClick={() => showToast('Toast message', 'toast-id', {
+      autoClose: true
+      color: 'danger'
+      iconName: 'download'
+    })}
+  >
+    Show Toast
+  </Button>
+
+  <UncontrolledToast alignmentX="right" alignmentY="top" closeLabel="Close toast" hasIcon isDismissible />
+</ToastProvider>
+```
+
+### useUncontrolledToast hook
+
+Hook that provides information on `UncontrolledToast` and functions for opening and closing.
+What is uncontrolled component you can find [here][react-uncontrolled]
+
+This hook returns:
+
+| Name        | Type                                                         | Default    | Description                                              |
```

Comment:
no required column?

Reply:
No, this is the table for `useUncontrolledToast` hook. 
So you can understand how to manage this component and what data you can retrieve from it.

Do we have a better way to pass that info to the developer?